pyface/ui/qt4/action/tool_bar_manager.py

```diff
@@ -55,6 +55,9 @@ class ToolBarManager(ActionManager):
     # Cache of tool images (scaled to the appropriate size).
     _image_cache = Instance(ImageCache)
 
+    #: Keep track of all created toolbars in order to properly dispose of them
+    _toolbars = []
```

Comment:
Same here. I am sorry for overlooking this.

Reply:
Opened #548